ideal_indirection.md

```diff
@@ -62,13 +73,13 @@ has occurred to the `base_addr` stored by the `page_table_entry`.
 
 ## Translation Lookaside Buffer (`tlb.c` and `tlb.h`)
 
-The Translation Lookaside Buffer will cache the base virtual address to the corresponding page table entry pointer.
+The Translation Lookaside Buffer will cache the base virtual address of a virtual page to the corresponding page table entry pointer.
 The implementation and header is provided to you.
 Make note of the use of double pointers.
 
-The reason why our TLB caches page table entry pointers instead of physical addresses of frames is because you will need to set metadata bits when translating addresses.
+The reason why our TLB caches page table entry pointers instead of physical addresses of frames is because you will need to set metadata bits in the `page_table_entry`s when translating addresses.
```

Comment:
```suggestion
Since the TLB caches page table entry pointers instead of physical addresses of frames, you will need to set metadata bits in the `page_table_entry`s when translating addresses.
```

Reply:
This changes the intended meaning of the sentence and is incorrect. Denied.